src/workload.rs

```diff
@@ -240,7 +327,7 @@ impl TryFrom<&XdsWorkload> for Workload {
                 resource.status,
             ))?,
 
-            native_hbone: resource.native_hbone,
+            native_hbone: resource.native_tunnel,
```

Comment:
nit: rename native_hbone in the internal representation as well

Reply:
addressed in https://github.com/istio/ztunnel/pull/486/commits/0d9ca6bfffa74f55f5af673d37c9592767df9087